lib/rules/indentation.js

```diff
@@ -2,22 +2,63 @@
 
 var helpers = require('../helpers');
 
+/**
+ * Check if a node is a newline character or not
+ *
+ * @param {Object} node - The node to test
+ * @returns {boolean} Whether the node is a newline or not
+ */
+var isNewLine = function (node) {
```

Comment:
This should probably be moved into the helpers file as it's pretty generic and we may need it again.